java/integTest/src/test/java/glide/SharedCommandTests.java

```diff
@@ -4973,6 +4973,218 @@ public void xgroupCreate_xgroupDestroy(BaseClient client) {
         assertInstanceOf(RequestException.class, executionException.getCause());
     }
 
+    @SneakyThrows
+    @ParameterizedTest(autoCloseArguments = false)
+    @MethodSource("getClients")
+    public void xinfoGroupsWithXinfoConsumers(BaseClient client) {
```

Comment:
```suggestion
    public void xinfoGroups_with_xinfo_consumers(BaseClient client) {
```
We have a fixture which replaces `_` by ` ` in the test report